### PR TITLE
Initial Admin setup

### PR DESCRIPTION
**Pull request by xandergroeneveld**

### CODE REVIEW

File: app/admin/farmer.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  
+  index do
+    selectable_column
+    id_column
+    column :email
+    column :last_name
+    column :first_name
+    column :business_name
+    column :location do |farmer|
+      farmer.location.label
```
Comment (groeneveld91) [inline app/admin/farmer.rb]: https://www.dropbox.com/s/tjhp1q0lbzgvzxl/Screenshot%202016-09-21%2018.00.38.png?dl=0

File: app/admin/farmer.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      f.input :contact_name
+      f.input :contact_number
+    end
+    f.actions
+  end
+
+  sidebar "Details", only: :show do
+    attributes_table_for farmer do
+      row :jobs do
+        link_to "#{farmer.jobs.length} listed jobs", :controller => "jobs", :action => "index", 'q[farmer_email_contains]' => "#{farmer.email}".html_safe
```
Comment (groeneveld91) [inline app/admin/farmer.rb]: do we want to use `q[farmer_email_equals]` here vs `q[farmer_email_contains]` ?

File: app/assets/javascripts/active_admin.js.coffee [hunk 3]
```diff
@@ -0,0 +1 @@
+#= require active_admin/base
```
Comment (groeneveld91) [inline app/assets/javascripts/active_admin.js.coffee]: do we have to use `.coffee` and coffee syntax here ? does active admin lock into that?

File: config/initializers/active_admin.rb [hunk 4]
```diff
@@ -0,0 +1,272 @@
+ActiveAdmin.setup do |config|
+  # == Site Title
+  #
+  # Set the title that is displayed on the main layout
+  # for each of the active admin pages.
+  #
+  config.site_title = "Agdraft"
```
Comment (groeneveld91) [inline config/initializers/active_admin.rb]: "AgDraft" i think

File: Gemfile [hunk 5]
```diff
@@ -37,6 +37,7 @@ gem "country_select"
 gem "stripe"
 gem "comfortable_mexican_sofa", "~> 1.12.0"
 gem "colorize"
+gem 'activeadmin', '~> 1.0.0.pre4'
```
Comment (xandergroeneveld) [inline Gemfile]: ah go fuck yourself

File: app/admin/dashboard.rb [hunk 6]
```diff
@@ -0,0 +1,52 @@
+ActiveAdmin.register_page "Dashboard" do
+
+  menu priority: 1, label: proc{ I18n.t("active_admin.dashboard") }
+
+  content title: proc{ I18n.t("active_admin.dashboard") } do
+    # Here is an example of a simple dashboard with columns and panels.
```
Comment (xandergroeneveld) [inline app/admin/dashboard.rb]: will remove

File: app/admin/farmer.rb (hunk 2 above)
Comment (xandergroeneveld) [inline app/admin/farmer.rb]: yeh good point

File: app/assets/javascripts/active_admin.js.coffee (hunk 3 above)
Comment (xandergroeneveld) [inline app/assets/javascripts/active_admin.js.coffee]: I don't think so. I will make it normal js.

**Pull request closed without merging**